File: source_directory/question-8-pass-0.py
# Maximize first array element by performing given operations at most K times. 
def getMax(arr, N, K): 
    # Initialize result 
    res = 0
    # Initialize count of operations 
    count = 0
    # Sort the array in increasing order 
    arr.sort() 
    # Traverse the array from left to right 
    for i in range(N): 
        # If the current element is smaller than the previous 
        # element, then increment the count of operations 
        if arr[i] < arr[i - 1]: 
            count += 1
        # If the count of operations exceeds K, then return 
        # the result 
        if count > K: 
            return res 
        # Update the result 
        res = max(res, arr[i]) 
    # Return the result 
    return res 
  
# Driver code 
arr = [1, 2, 3, 4, 5] 
N = len(arr) 
K = 2
print("Maximum element in the array is", getMax(arr, N, K)) 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by Rajat Mishra 

# This code is contributed by

File: results/preliminary_analysis/baseline_cpp_results/cpp_source/question-8-pass-0.cpp
// Maximize first array element by performing given operations at most K times. 
 #include <bits/stdc++.h> 
 using namespace std ;
 int getMax ( int arr [ ] , int N , int K ) {
 
 	int max = arr [ 0 ] ;
 	int i = 0 ;
 	int j = 1 ;
 	int count = 0 ;
 	while ( count < K ) {
 		if ( arr [ i ] < arr [ j ] ) {
 			max = max + arr [ j ] ;
 			count++ ;
 			j++ ;
 			if ( j == N ) {
 				j = 0 ;
 			}
 		}
 		else {
 			i++ ;
 			if ( i == N ) {
 				i = 0 ;
 			}
 		}
 	}
 	return max ;
 }
 
 int main ( ) {
 	int T ;
 	cin >> T ;
 	while ( T-- ) {
 		int N ;
 		cin >> N ;
 		int arr [ N ] ;
 		for ( int i = 0 ; i < N ; i++ ) {
 			cin >> arr [ i ] ;
 		}
 		